static/src/stylesheets/base/_inline-svgs.scss

```diff
@@ -52,7 +52,7 @@
     }
 
     .tonal--tone-special-report & {
-        fill: $news-support-6;
+        fill: #ffffff;
```

Comment:
could we use a variable? [`neutral-7`](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/palette/_src.scss#L15) perhaps? or a declare a new one?